src/LiquidityPool.sol

```diff
@@ -170,6 +181,13 @@ contract LiquidityPool is Auth {
         investmentManager.requestRedeem(shares, owner);
     }
 
+    function requestRedeemWithPermit(uint256 shares, address owner, uint256 deadline, uint8 v, bytes32 r, bytes32 s)
```

Comment:
Likewise, it's not immediately obvious to me what v,r, and s should be, or where I can go to find that out. Since it's a public function, I think a quick explainer in a `@dev` comment would be nice